getting_started/step_by_step/signals.rst

```diff
@@ -255,6 +300,14 @@ We can now connect the Timer to the Sprite2D in the ``_ready()`` function.
         var timer = get_node("Timer")
         timer.timeout.connect(_on_Timer_timeout)
 
+ .. code-tab:: csharp C#
+
+    public override void _Ready()
+    {
+        var timer = GetNode<Timer>("Timer");
+        timer.Connect("timeout", this, "OnTimerTimeout");
```

Comment:
We recommend using `nameof` for the method name as it's less error-prone.

```suggestion
        timer.Connect("timeout", this, nameof(OnTimerTimeout));
```

Reply:
done